packages/pds/src/context.ts

```diff
@@ -256,12 +263,36 @@ export class AppContext {
       appviewCdnUrlPattern: cfg.bskyAppView?.cdnUrlPattern,
     })
 
+    const safeAgent = new undici.Agent({
+      allowH2: cfg.fetch.allowHTTP2, // This is experimental
+      headersTimeout: cfg.fetch.headersTimeout,
+      maxResponseSize: cfg.fetch.maxResponseSize,
```

Comment:
are we cool re-using these config values for both the safeFetch & safeAgent? 

only issue I could think of is if we have different tradeoffs on `maxResponseSize`